docs/upgrade/_common/upgrade-guide-v2022-patch-ubuntu-and-debian-p2.rst

```diff
@@ -0,0 +1,95 @@
+**To upgrade ScyllaDB:**
+
+#. Update the |APT|_ to **2022.x**.
+#. Install:
+
+    .. code:: sh
+
+       sudo apt-get clean all
+       sudo apt-get update
+       sudo apt-get dist-upgrade scylla-enterprise
```

Comment:
These step is relevant to the old/previous procedure

Reply:
I think this step should be left as is. I was told that this is an alternative procedure - not recommended but should be included. And this is what was specified in the introduction: 

There are two alternative upgrade procedures:

- [Upgrading ScyllaDB and simultaneously updating 3rd party and OS packages](http://127.0.0.1:5500/upgrade/upgrade-enterprise/upgrade-guide-from-2022.x.y-to-2022.x.z/upgrade-guide-from-2022.x.y-to-2022.x.z-image/#upgrade-image-recommended-procedure). It is recommended if you are running a ScyllaDB official image (EC2 AMI, GCP, and Azure images), which is based on Ubuntu 20.04.
- [Upgrading ScyllaDB without updating any external packages](http://127.0.0.1:5500/upgrade/upgrade-enterprise/upgrade-guide-from-2022.x.y-to-2022.x.z/upgrade-guide-from-2022.x.y-to-2022.x.z-image/#upgrade-image-enterprise-upgrade-guide-regular-procedure).